### ISSUE TITLE
Screen cause client crash

### ISSUE THREAD

**Issue by Nagispace** [McJtyMods/RFToolsBase#27]
Here is the crash report, only crash client as soon as the player load the chunk that has the problematic screen.

[crash-2020-11-19_17.43.24-client.txt](https://github.com/McJtyMods/RFToolsBase/files/5568372/crash-2020-11-19_17.43.24-client.txt)

**Comment by McJty** [comment]
Update

**Comment by Nagispace** [comment]
> 
> 
> Update

Which mods should I update ?
I updated : 
- rftoolsbuilder to version 1.16-3.0.10
- rftoolsutility to version 1.16-3.0.18
- rftoolsbase to version 1.16-2.0.7

The client still crash
[Uploading crash-2020-11-20_14.23.30-client.txt…]()

**Comment by McJty** [comment]
You also need to update mcjtylib but there is still a bug. Others have reported it to me

**Comment by Nagispace** [comment]
> 
> 
> You also need to update mcjtylib but there is still a bug. Others have reported it to me

Is there any workaround ?
Not being able to go into my base is pretty annoying and even with depleted batteries the screen still make the game crash

**Comment by McJty** [comment]
Yes, there is. Provided you are running the latest versions of all my mods you can edit rftoolsutility-client.toml and modify this line:

	#Set to true for force TrueType to be disabled in all cases. Use this in case the truetype font is causing issues
	forceNoTruetype = false

Change it to true to force no truetype and that should fix it
